common/middleware.js

```diff
@@ -0,0 +1,24 @@
+var jwt = require("jsonwebtoken");
+const jwtSecret = "ghdsjvbiuy43870yoh22#$hoihnAkpvbjhTN)3&_N";

```

Comment:
Please remove this secret from this file as it can cause security leak. Consider using a .env file